spec/bundler/ruby_dsl_spec.rb

```diff
@@ -0,0 +1,93 @@
+require "spec_helper"
```

Comment:
Think this might be missing the `# frozen_string_literal: true` line at the top? Travis seems to be throwing errors/failing out whenever a spec or file is missing this.